ExternalRevisions/SPIRV-Cross_repo_revision

```diff
@@ -1 +1 @@
-d9ed3dcc7a7e62e5f95fd8f24e3d35e7e402ae92
+e37fbc6f72495e90597f6caddb4c9e309029df38
```

Comment:
`fetchDependencies` build generating numerous new warnings from SPIRV-Cross of this type:

```
External/SPIRV-Cross/spirv_msl.cpp:1898:43: warning:
      implicit conversion loses integer precision: 'std::__1::vector<unsigned int,
      std::__1::allocator<unsigned int> >::size_type' (aka 'unsigned long') to 'uint32_t'
      (aka 'unsigned int') [-Wshorten-64-to-32]
                uint32_t mbr_idx = ib_type.member_types.size();
                         ~~~~~~~   ~~~~~~~~~~~~~~~~~~~~~^~~~~~
```

Reply:
Just need to wait for KhronosGroup/SPIRV-Cross#870 to be merged...